core/templates/components/brand-bar/brand-bar.twig

```diff
@@ -0,0 +1,3 @@
+<section{{ attributes }} id="su-brand-bar" class="{{ modifier_class }}">
+  <div class="su-brand-bar__container"><a{{ attributes }} class="su-brand-bar__logo" href="https://stanford.edu">Stanford University</a></div>
```

Comment:
I think you can remove the attributes variable from the `<a>` tag or at the very least, change the variable name so that the `<section>` wrapper attributes do not get applied to the `<a>` tag.

Reply:
Sure, will do. I mistook the {{attributes}} tag as a generic "the site builder can add any attribute here" tag.

Reply:
only one attributes variable can be sent to this twig file so it should not be used on more than one html element unless you intend to have exactly the same properties.

Reply:
Yup, I've removed it from `<section>` so only `<a>` has the {{ attributes }} variable now. Thanks.